mava/components/building/best_checkpointer.py

```diff
@@ -6,15 +7,26 @@
 
 from mava.components.component import Component
 from mava.core_jax import SystemBuilder, SystemParameterServer
+from mava.utils.lp_utils import termination_fn
 
 
 @dataclass
 class BestCheckpointerConfig:
     checkpointing_metric: Tuple = ("mean_episode_return",)
     checkpoint_best_perf: bool = False
+    # Flag to calculate the absolute metric
+    absolute_metric: bool = False
+    # How many episodes to run evaluation for
+    absolute_metric_duration: Optional[Any] = None
```

Comment:
Nice addition with the comments! Maybe add them for the other two config options also

Reply:
Done!